admin/python/lsst/qserv/admin/dataDuplicator.py

```diff
@@ -68,30 +69,34 @@ def _runIndex(self):
         """
 
         for table in self._tables:
-            if os.path.isfile(os.path.join(self._cfgDirname, table + '.cfg'))==False:
+            if os.path.isfile(os.path.join(self._cfgDirname, table + '.cfg')) == False:
                 self.logger.error("Path to indexing config file not found")
 
             self.logger.info("Running indexer with output for %r to %r" % (table, self._outDirname))
             run_index = commons.run_command(["sph-htm-index",
-                                             "--config-file=" + os.path.join(self._cfgDirname, table + ".cfg"),
+                                             "--config-file=" +
+                                             os.path.join(self._cfgDirname, table + ".cfg"),
                                              "--config-file=" + os.path.join(self._cfgDirname, "common.cfg"),
                                              "--in=" + os.path.join(self._cfgDirname, table + ".txt"),
                                              "--out.dir=" + os.path.join(self._outDirname, "index/", table)])
 
-
     def _runDuplicate(self):
         """
         Run sph-duplicate to set up partitioned data that the data loader needs
         """
 
         for table in self._tables:
-            if os.path.isfile(os.path.join(self._cfgDirname, 'common.cfg'))==False:
+            if os.path.isfile(os.path.join(self._cfgDirname, 'common.cfg')) == False:
                 self.logger.error("Path to duplicator config file not found")
 
             self.logger.info("Running duplicator for table %r" % table)
             run_dupl = commons.run_command(["sph-duplicate",
                                             "--config-file=" + os.path.join(self._cfgDirname, table + ".cfg"),
                                             "--config-file=" + os.path.join(self._cfgDirname, "common.cfg"),
-                                            "--index=" + os.path.join(self._outDirname, "index/", table, "htm_index.bin"),
-                                            "--part.index=" + os.path.join(self._outDirname, "index", self._directorTable, "htm_index.bin"),
+                                            "--index=" +
+                                            os.path.join(
+                                            self._outDirname, "index/", table, "htm_index.bin"),
+                                            "--part.index=" +
+                                            os.path.join(
+                                            self._outDirname, "index", self._directorTable, "htm_index.bin"),
```

Comment:
This looks a big ugly, I think function parameters want an indent (and three lines above too).